civihr_default_theme/scss/base/_variables.scss

```diff
@@ -163,3 +163,8 @@ $crm-scrollbar-thumb-color: $crm-gray-light-blue !default;
 $crm-customselect-height:        33 !default;
 $crm-customselect-caret-width:   32 !default;
 $crm-customselect-padding-right: $crm-customselect-caret-width + 12 !default;
+
+//== Forms
+//
+//##
+$input-border:              $crm-color-neutral !default;
```

Comment:
Yes it is bootstrap, other components were using `$crm-color-neutral` for the border, so replaced it with `$input-border`. Added in PR document too.